pkg/webhook/controlplane/ensurer.go

```diff
@@ -76,23 +76,44 @@ var (
 			},
 		},
 	}
+	// config mount for auditpolicy-webhook-config that is specified at kube-apiserver commandline
+	auditpolicyWebhookConfigVolumeMount = corev1.VolumeMount{
+		Name:      metal.AuditpolicyWebHookConfigName,
+		MountPath: "/etc/webhook/config",
```

Comment:
This is a problem:
The Pod "configmap-pod" is invalid: spec.containers[0].volumeMounts[1].mountPath: Invalid value: "/etc/config": must be unique